README.md

```diff
@@ -136,19 +136,25 @@ tracer->report(std::move(tracing_context));
 #### Skywalking CDS
 
 C++ agent implements Skywalking CDS feature it allows to change bootstrap config dynamically from the response of sync request, invoked from this periodically.
+Dynamically configurable values are described in description of properties on `docs/README.md'.
 
 ```cpp
 TracerConfig config;
 // If you set this value as zero, CDS request won't occur.
 config.set_cds_request_interval(5); // CDS request interval should be 5sec
 ```
 
-Currently, Configurable values dynamically are like below.
+If you are using Consul KVS as backend, we could put configuration value through HTTP request.
 
-| Config Key | Value Description |
-|:----:|:----:|
-|instance_name| Instance name of this agent |
+```config.yaml
```

Comment:
I think `config.yaml` doesn't show up on the GitHub preview page, https://github.com/SkyAPM/cpp2sky/blob/0d8bb7949686c8c6c6c3baba188a7c6b18be2a5e/README.md